src/Illuminate/Pipeline/Pipeline.php

```diff
@@ -248,4 +248,21 @@ protected function handleException($passable, Exception $e)
     {
         throw $e;
     }
+
+    /**
+     * Creates a new Pipeline instance.
+     *
+     * @param  mixed $passable
+     * @return static
+     */
+    public static function make($passable = null)
+    {
+        $instance = app(static::class);
```

Comment:
Thanks, but we can't accept this code because of this. The pipeline component can't call code within the foundation component. This is why we accept a container contract in the constructor.

Reply:
Got it. I think the `app(Pipeline::class)` is here to stay.